Gordon360/Controllers/NewsController.cs

```diff
@@ -44,7 +46,12 @@ public ActionResult<StudentNewsViewModel> GetByID(int newsID)
                 return NotFound();
             }
 
-            result.Image = ImageUtils.RetrieveImageFromPath(result.Image);
+            if (result.Image != null)
+            {
+                var imagePath = Path.Combine(_webHostEnvironment.ContentRootPath, "browseable", "uploads", Path.GetFileName(result.Image));
+
+                result.Image = ImageUtils.RetrieveImageFromPath(imagePath);
+            }
```

Comment:
Since we are now saving the public URL of the image to the database, we don't want to send base64 image data to the frontend. Instead, we can just send the public URL of the image, and let the browser fetch the image itself. That will be much more performant.

That means you don't need to do anything the the Image column here, just pass it as is to the frontend.